pkgs/development/python-modules/pymeta3/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+}:
+let
+  properCaseName = "PyMeta3";
+in
+buildPythonPackage rec {
+  pname = lib.toLower properCaseName;
+  version = "0.5.1";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = properCaseName;
+    hash = "sha256-GL2jJtmpu/WHv8DuC8loZJZNeLBnKIvPVdTZhoHQW8s=";
+  };
+
+  doCheck = false; # Tests do not support Python3
```

Comment:
:(

Reply:
:(

(marking this resolved just to clean up the PR. please reopen if there's something you'd like us to do here)

Reply:
Not a big fan of adding already unmaintained software